src/Symfony/Bridge/PhpUnit/Extension/RegisterClockMockSubscriber.php

```diff
@@ -34,6 +41,10 @@ public function notify(Loaded $event): void
                     ClockMock::register($test->className());
                 }
             }
+
+            foreach ($this->reader->forClassAndMethod($test->className(), $test->methodName(), TimeSensitive::class) as $attribute) {
+                ClockMock::register($attribute->class);
```

Comment:
Why do we require the class to be set when we could use $test->className() instead?

Reply:
This is for cases where the namespace is different from the class being tested, like a dependency of the tested class. But I see your point, maybe the class can be optional, eg:

```php
ClockMock::register($attribute->class ?? $test->className());
```

Reply:
Done, https://github.com/symfony/symfony/compare/54c59f7b5436f539021fa00fa2cc67dedc77ceac..42cc6d2a497c80507496ba46fc80917668503e12